services/tilesets.js

```diff
@@ -16,6 +16,11 @@ var Tilesets = {};
  *
  * @param {Object} [config]
  * @param {string} [config.ownerId]
+ * @param {Object} [query]
+ * @param {number} [query.range]
+ * @param {string} [query.sortBy]
+ * @param {string}  [query.start]
```

Comment:
In keeping with other parts of this library, like https://github.com/mapbox/mapbox-sdk-js/blob/master/services/styles.js#L175 let's include a description of these fields here.